presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2285,6 +2286,29 @@ public void testCreateExternalTableTextFileFieldSeparatorEscape()
         deleteRecursively(tempDir.toPath(), ALLOW_INSECURE);
     }
 
+    @Test
+    public void testCreateExternalTableWithCTASWritesDisabled()
+            throws IOException
+    {
+        File tempDir = createTempDir();
+        File dataFile = new File(tempDir, "test.txt");
+
+        @Language("SQL") String createTableSql = format("" +
+                        "CREATE TABLE %s.%s.test_create_external " +
```

Comment:
I think `%s.%s.` is redundant -- since this is session's default, it's implict in SQL.

(i know other tests are doing this, but... hm .... let's not do this here)